bookserver/crud.py

```diff
@@ -35,13 +35,15 @@
     CodeValidator,
     CourseInstructor,
     CourseInstructorValidator,
+    UserSubChapterProgres,
```

Comment:
Should be spelled progress.

Reply:
Huh, the autogenerator tool seems to have dropped the second  s for some reason.